lib/v2/datasource/remote/api/network_repository.dart

```diff
@@ -24,7 +24,6 @@ abstract class NetworkRepository {
   String table_recover = 'recovers';
 
   Result mapHttpResponse(http.Response response, Function modelMapper) {
-    print('mapHttpResponse - statusCode: ' + response.statusCode.toString());
     switch (response.statusCode) {
```

Comment:
that printout is redundant - we should really only print on error, otherwise it's way too much in the logs, and mostly we don't need to know about success

same as unix commands.